src/Policies/FeedbackPolicy.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tipoff\Feedback\Policies;
+
+use Illuminate\Auth\Access\HandlesAuthorization;
+use Tipoff\Feedback\Models\Feedback;
+use Tipoff\Support\Contracts\Models\UserInterface;
+
+class FeedbackPolicy
+{
+    use HandlesAuthorization;
+
+    /**
+     * Determine whether the user can view any models.
+     *
+     * @param  \App\Models\User  $user
```

Comment:
The @param for psalm needs to be updated from `\App\Models\User` to `Tipoff\Support\Contracts\Models\UserInterface`

Reply:
@devjk1 let me know I this works after you test it again.